extension/table.go

```diff
@@ -212,9 +283,20 @@ func (r *TableHTMLRenderer) renderTableCell(w util.BufWriter, source []byte, nod
 	if entering {
 		align := ""
 		if n.Alignment != ast.AlignNone {
-			align = fmt.Sprintf(` align="%s"`, n.Alignment.String())
+			if _, ok := n.AttributeString("align"); !ok { // Skip align render if overridden
+				// TODO: "align" is deprecated. style="text-align:%s" instead?
+				align = fmt.Sprintf(` align="%s"`, n.Alignment.String())
+			}
+		}
+		fmt.Fprintf(w, "<%s", tag)
+		if n.Attributes() != nil {
+			if tag == "td" {
+				html.RenderAttributes(w, n, TableTdCellAttributeFilter) // <td>
```

Comment:
It seems that this line renders an `align` attribute because `TableTdCellAttributeFilter` includes `align` . But `align` is also rendered at L299. Right?

Reply:
if n.AttributeString("align") does not exist, we generate **align** string. If it exists, then we skip fmt.Fprintf(w, "%s>", align) will print `>` only then.